isaric/parsers/ccp-senegal.toml

```diff
@@ -244,8 +244,14 @@
   field = "healthwork_erterm"
   ref = "Y/N/NK"
 
+[subject.works_lab]
+  field = "labwork_erterm"
+  description = "employed in a biological analysis laboratory?"
+  ref = "Y/N/NK"
```

Comment:
@pipliggins I'm thinking perhaps this variable should only map to works_lab, as it is not specifically asking about micro lab work